zerver/decorator.py

```diff
@@ -308,8 +310,11 @@ def access_user_by_api_key(
 def log_exception_to_webhook_logger(
     summary: str,
     unsupported_event: bool,
+    anomalous_payload: Optional[bool] = False,
 ) -> None:
-    if unsupported_event:
+    if anomalous_payload:
+        webhook_anomalous_payloads_logger.exception(summary, stack_info=True)
+    elif unsupported_event:
         webhook_unsupported_events_logger.exception(summary, stack_info=True)
     else:
         webhook_logger.exception(summary, stack_info=True)
```

Comment:
I'm not thrilled with what is really a tri-state value being expressed as two `bool`s.  Can we just pass down the `err` exception object, rather than its `str` and a bunch of bools?

Reply:
Good point! Thanks! :)